### ISSUE TITLE
Use bumpversion to version files based on tag

### ISSUE THREAD

**Issue by ericwb** [vmware/precaution#110]
**Is your feature request related to a problem? Please describe.**
Once we start releasing versioned tags, we should take advantage of bumpversion to automatically update files with a version number in them.  Currently `package.json` has a version, but more files have one in the future.

**Describe the solution you'd like**
Use bumpversion

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

**Comment by evqna** [comment]
Are you specifically referencing the [Python tool](https://github.com/peritus/bumpversion) or any similar alternative?

**Comment by evqna** [comment]
Apparently the npm CLI has a commend just for this: https://docs.npmjs.com/cli/version.html

**Comment by evqna** [comment]
Same with Yarn: https://yarnpkg.com/lang/en/docs/cli/version/

**Comment by ericwb** [comment]
Yep, since we only have a version in the package.json currently, might as well use the npm command.
